CONTRIBUTING.md

```diff
@@ -129,3 +129,7 @@ See the [code owners](CODEOWNERS) file.
 
 See the [community membership document in OpenTelemetry community
 repo](https://github.com/open-telemetry/community/blob/master/community-membership.md).
+
+## FAQ
+### Where should I put third party propagators/exporters, contrib or standalone crates?
```

Comment:
wanted to clarify the problem that where to put the stuff here. Please let me know if the below descriptions makes sense. @jtescher

Reply:
Seems good 👍